zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -36,6 +36,8 @@ import {
   FormGroup,
   InputBase,
   TextField,
+  Tooltip,
+  Button,
```

Comment:
Oh I see we added a button why was that?

Reply:
😅 yes sure i will remove it